code/game/objects/items/dehy_carp.dm

```diff
@@ -6,69 +6,68 @@
 //Child of carpplushie because this should do everything the toy does and more
 /obj/item/toy/plush/carpplushie/dehy_carp
 	var/mob/owner = null //Carp doesn't attack owner, set when using in hand
-	var/owned = 0 //Boolean, no owner to begin with
 	var/mobtype = /mob/living/simple_animal/hostile/carp //So admins can change what mob spawns via var fuckery
+	var/swelling = FALSE
 
 //Attack self
 /obj/item/toy/plush/carpplushie/dehy_carp/attack_self(mob/user)
-	src.add_fingerprint(user) //Anyone can add their fingerprints to it with this
-	if(!owned)
-		to_chat(user, span_notice("You pet [src]. You swear it looks up at you."))
-		owner = user
-		owned = 1
-	else
+	if(owner)
 		return ..()
+	add_fingerprint(user)
```

Comment:
wtf balance change (prints are only from the first person to attack self with it)

Reply:
the parent calls interact which calls add fingerprint